docs/rootfs-and-kernel-setup.md

```diff
@@ -59,14 +59,14 @@ The kernel images used in our CI to test Firecracker's features are obtained by
 using the recipe inside devtool:
 
 ```bash
-config="resources/guest_configs/microvm-kernel-x86_64-4.14.config"
+config="resources/guest_configs/microvm-kernel-x86_64-5.10.config"
```

Comment:
Mh, this entire "build_kernel" command doesn't exist anymore :joy: The only way to build the kernels seems to be via `resources/rebuild.sh` these days.

Reply:
pffff, you're right. I was just greping and changing. How do you suggest we handle this?

Reply:
I guess just ignoring it for now is fine, we should probably take note internally to update this though haha